core/src/main/java/org/apache/carbondata/core/datamap/status/DataMapStatusManager.java

```diff
@@ -142,13 +142,13 @@ public static DataMapSchema getDataMapSchema(String dataMapName)
   }
 
   /**
-   * This method will remove all segments of dataMap table in case of Insert-Overwrite/Update/Delete
+   * This method will remove all segments of MV table in case of Insert-Overwrite/Update/Delete
    * operations on main table
    *
    * @param allDataMapSchemas of main carbon table
    * @throws IOException
    */
-  public static void truncateDataMap(List<DataMapSchema> allDataMapSchemas)
+  public static void truncateMVTable(List<DataMapSchema> allDataMapSchemas)
```

Comment:
DataMapSchema can not be changed in this PR since Index and MV still both use it. Metadata unification will be done in another PR.
I think @niuge01  is doing it

Reply:
ok